web/src/portico/team.js

```diff
@@ -115,7 +115,7 @@ export default function render_tabs() {
                             name: get_display_name(c),
                             github_username: c.github_username,
                             avatar: c.avatar,
-                            profile_url: get_profile_url(c),
+                            profile_url: get_profile_url(c, tab_name),
```

Comment:
I'll fix this when merging this time, but this commit message should explain the actual problem -- what this parameter does, and thus why it's correct to add it.